src/main/java/de/tum/in/www1/artemis/repository/TeamRepository.java

```diff
@@ -20,6 +20,9 @@
 
     List<Team> findAllByExerciseCourseIdAndShortName(@Param("courseId") Long courseId, @Param("shortName") String shortName);
 
+    @Query("select count(distinct team) from Team team where team.exercise.id = :#{#exerciseId}")
+    Integer getAmountByExerciseId(@Param("exerciseId") Long exerciseId);
```

Comment:
```suggestion
    Integer getNumberOfTeamsForExercise(@Param("exerciseId") Long exerciseId);
```
Maybe something like this would be more clearer :)?

Reply:
Sounds better :)